### PR TITLE
UIBULKED-389 Add Suppress from discovery option to Instance records edits

### PR DESCRIPTION
**Pull request by vashjs**
After merging this PR, we will have Suppress from discovery option for Instances. After selecting one of the action (true/false), 2 additional checkboxes will be displayed. See GIF below. Ref: [UIBULKED-389](https://issues.folio.org/browse/UIBULKED-389)

UIBULKED-389

### CODE REVIEW

**Review by UladzislauKutarkin — APPROVED** [review]

File: src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```
Comment (BogdanDenis) [inline src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.js:1]: Please re-order imports to be:
1. Third-party libraries
2. Folio libraries
3. Local imports

And leave an empty line between each group

File: src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { Checkbox, Col } from '@folio/stripes/components';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { useDerivativeModification } from '../../../../../hooks';
+import { ACTION_PARAMETERS_KEY } from './helpers';
+import css from '../BulkEditInApp.css';
+
+export const AdditionalActionParameters = ({ action, actionIndex, onChange }) => {
+  const { formatMessage } = useIntl();
+  const dynamicWidth = action.parameters * 2;
```
Comment (BogdanDenis) [inline src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.js:11]: If `action` prop is not required, then this should be `action?.parameters * 2`. Please check prop types and how other props are accessed

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.test.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```
Comment (artem-blazhko) [inline src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.test.js:1]: It is not necessary here

File: src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/helpers.test.js [hunk 4]
```diff
@@ -2,10 +2,11 @@ import {
   ACTIONS,
   CONTROL_TYPES,
   OPTIONS,
-  commonAdditionalActions,
+  commonAdditionalActions, CAPABILITIES, PARAMETERS_KEYS,
```
Comment (artem-blazhko) [inline src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/helpers.test.js:5]:
```suggestion
  commonAdditionalActions,
  CAPABILITIES,
  PARAMETERS_KEYS,
```

File: src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/helpers.test.js [hunk 5]
```diff
@@ -738,7 +861,12 @@ describe('ContentUpdatesForm helpers', () => {
       });
 
       it('returns the correct object for the default case', () => {
-        expect(getDefaultActions('unknown', [], formatMessage))
+        expect(getDefaultActions({
+          option: 'unknown',
```
Comment (artem-blazhko) [inline src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/helpers.test.js:865]: Should we create constant for `unknown` ?

Comment (vashjs) [reply src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/helpers.test.js:867]: Thanks for comment, not sure that it's required for this case, I'm using it only once in this test file

File: src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.test.js (hunk 3 above)
Comment (vashjs) [reply src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.test.js:1]: thanks, removed

File: src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.js (hunk 2 above)
Comment (vashjs) [reply src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.js:13]: thanks, fixed

File: src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.js (hunk 1 above)
Comment (vashjs) [reply src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/AdditionalActionParameters.js:1]: thanks, fixed

**Comment by vashjs** [comment]
@BogdanDenis  @artem-blazhko thanks for the comments. all was addressed. Please double check

**Review by UladzislauKutarkin — APPROVED** [review]

**Review by BogdanDenis — APPROVED** [review]

**Review by artem-blazhko — APPROVED** [review]

**Pull request merged**